libbeat/cfgfile/reload_integration_test.go

```diff
@@ -0,0 +1,123 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
This file was actually renamed from `reload_test.go`. However because a test is added to `reload_test.go`, Git got confused and thinks this file is new and the other has been edited.